setup.py

```diff
@@ -36,7 +36,8 @@
         ],
         'migrators': [
             'iso = pulp_2to3_migration.app.plugin.iso.migrator:IsoMigrator',
-            'docker = pulp_2to3_migration.app.plugin.docker.migrator:DockerMigrator',
+#            'docker = pulp_2to3_migration.app.plugin.docker.migrator:DockerMigrator',
```

Comment:
what's is the reason this is commented out? please make sure it is uncommented when you merge

Reply:
It's commented out because it doesn't work for the pulpcore master branch (does_batch issue) + it was not possible to install it at some point.
Actually, maybe now it's installable since the pulpcore version in pulp_container has been fixed.

Reply:
updated